src/questions/fjord-foundry.yaml

```diff
@@ -0,0 +1,170 @@
+- uuid: fed2ef38-e906-4e1b-93bf-d4ec5bdf3ac3
+  type: SingleChoice
+  content: What is Fjord Foundry?
+  hint: NoHint
+  explanation: Fjord Foundry is a community platform that connects project teams with passionate supporters via Liquidity Bootstrapping Pools (LBP's). It empowers project teams to generate liquidity directly through community participation.
+  answerKeys:
+    - B
+  difficultyLevel: Low
+  choices:
+    - content: A platform that raises money for individuals
+      key: A
+    - content: A platform that empowers project teams to generate liquidity directly through community participation
+      key: B
+    - content: A platform for money exchange
+      key: C
+    - content: A platform that creates sniper bots
+      key: D
+- uuid: 863ed099-ee12-4ca3-bca6-4a8bf8ef10d6
+  type: SingleChoice
+  content: How should the starting price of an LBP be set in comparison to what is believed to be the reasonable price?
+  hint: NoHint
+  explanation: Consider the beginning price of your LBP to be the ceiling you wish to set for the token sale. This may appear contradictory, but because LBPs operate differently than other token sales, your starting price should be significantly higher than what you believe to be the reasonable price. This does not imply that you are attempting to sell the token for more than it is worth.
+  answerKeys:
+    - B
+  difficultyLevel: Low
+  choices:
+    - content: The same
+      key: A
+    - content: Significantly higher
+      key: B
+    - content: Comparable
+      key: C
+    - content: Significantly lower
+      key: D
+- uuid: a63726bc-1e36-4d9a-97b8-0fc8b405fac7
+  type: SingleChoice
+  content: Who chooses the starting and ending weights and timings in a Liquidity Bootstrapping Pool (LBP)?
+  hint: NoHint
+  explanation: The pool owner, who also has the ability to pause exchanges, chooses the starting and ending weights and timings.
+  answerKeys:
+    - A
+  difficultyLevel: Low
+  choices:
+    - content: The pool owner
+      key: A
+    - content: The pool participant
+      key: B
+    - content: The pool user
+      key: C
+    - content: The pool buyer
+      key: D
+- uuid: 2f1ea1ea-2bc3-403d-8caa-33155fc9403e
+  type: SingleChoice
+  content: What is the goal of setting a high initial price in a Liquidity Bootstrapping Pool (LBP)?
+  hint: NoHint
+  explanation: Setting a high initial price allows the changing pool weights of your LBP to fully effect the price, gradually dropping it until market equilibrium is established. Unlike previous token sale methods, such as bonding curves, consumers are disincentivized to buy early and instead gain from waiting for the price to fall until it hits a level they consider fair.
+  answerKeys:
+    - A
+  difficultyLevel: Low
+  choices:
+    - content: To sell the token for more than it is worth
```

Comment:
Not sure if this is a right choice.

"Consumers are disincentivized to buy early, so it prevents majority of tokens getting sold to whales, thus helping in uniform distribution of the tokens and creating and effective community"

Reply:
Oh that's a mistake, I marked the wrong option as answer. Correct answer should be option B.